commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2ImportSubCommand.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.commandline.subcommands;
+
+import static tech.pegasys.web3signer.slashingprotection.SlashingProtectionFactory.createSlashingProtection;
+
+import tech.pegasys.web3signer.core.InitializationException;
+import tech.pegasys.web3signer.slashingprotection.SlashingProtection;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.HelpCommand;
+import picocli.CommandLine.MissingParameterException;
+import picocli.CommandLine.Model.CommandSpec;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Spec;
+
+@Command(
+    name = "import",
+    description = "Exports the slashing protection database",
+    subcommands = {HelpCommand.class},
+    mixinStandardHelpOptions = true)
+public class Eth2ImportSubCommand implements Runnable {
+
+  @Spec private CommandSpec spec;
+
+  @CommandLine.ParentCommand private Eth2SubCommand eth2Config;
+
+  @Option(
+      names = "--from",
+      description =
+          "The file from which the slashing protection database is to be imported. File is in interchange format")
+  File from;
+
+  @Override
+  public void run() {
+    if (from == null) {
+      throw new MissingParameterException(
+          spec.commandLine(), spec.findOption("--from"), "--from has not been specified");
+    } else if (eth2Config.slashingProtectionDbUrl == null) {
+      throw new MissingParameterException(
+          spec.parent().commandLine(),
+          spec.findOption("--slashing-protection-db-url"),
+          "--slashing-protection-db-url has not been specified");
+    }
+
+    final SlashingProtection slashingProtection =
+        createSlashingProtection(
+            eth2Config.slashingProtectionDbUrl,
+            eth2Config.slashingProtectionDbUsername,
+            eth2Config.slashingProtectionDbPassword);
+
+    try (final InputStream inStream = new FileInputStream(from)) {
+      slashingProtection.importData(inStream);
+    } catch (final IOException e) {
+      throw new RuntimeException("Unable to find input file", e);
```

Comment:
nit: UncheckedIOException is probably better suited to wrap IOException. Also, the IO exception may not necessary due to unable to find it. It can be permission issues or any other related problem.

Reply:
done